release-notes/major9/09.19.0/valtimo-frontend-libraries.md

```diff
@@ -13,6 +13,33 @@ The following features were added:
 
   Various tooltips for plugin configuration components have been added or improved.
 
+* **Improved the select component of the @valtimo/user-interface library**
```

Comment:
I think this is a bit more detailed than necessary. Is it possible to combine these three sub bullets into one small section? 'Contrast and color updates' feels like it describes what 'Background color changes' already describes, and the last bullet in particular can just be summed up with what is already written as the bullet itself (so hanged and created new color variables for the `v-select` element).

Reply:
I removed the extract topics and tried to make it smaller and more descriptive